client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -173,7 +174,7 @@ class JetpackSiteStats extends Component {
 					</FormFieldset>
 				</FoldableCard>
 
-				<CompactCard href={ '/stats/day/' + siteId }>
+				<CompactCard href={ getStatsPathForTab( 'day', siteId ) }>
```

Comment:
Nice, done in https://github.com/Automattic/wp-calypso/pull/13407/commits/36177f3f87646afa26607ec9db7c1678637b1d1f.